galleon-pack/src/main/resources/feature_groups/distributable-web.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="distributable-web" xmlns="urn:jboss:galleon:feature-group:1.0">
+
+    <feature spec="subsystem.distributable-web">
+        <param name="default-session-management" value="default"/>
+        <param name="default-single-sign-on-management" value="default"/>
+        <feature spec="subsystem.distributable-web.infinispan-session-management">
+            <param name="infinispan-session-management" value="default"/>
+            <param name="cache-container" value="web"/>
+            <param name="granularity" value="SESSION"/>
+            <feature spec="subsystem.distributable-web.infinispan-session-management.affinity.primary-owner"/>
```

Comment:
We probably want a separate non-clustered version of this feature-group-spec configured to use spec="subsystem.distributable-web.infinispan-session-management.affinity.local".
While "primary-owner" will fallback to equivalent behavior, it is likely going to be confusing.

Reply:
@pferraro What would that feature-group be called?

Reply:
distributable-web-local?